class1/README.md

```diff
@@ -1 +1,131 @@
 #Class 1
+
+##Goals: 
+- Students should understand course plan and resources
+- Development environments ready
+- Fundamentals of Node and JavaScript
+
+##Homework Due:
+For this first class you should:
+- Have a working instance of linux (probably Ubuntu). Mac users will probably be able to get by with OS X. Note that we will officially be doing everything on Ubuntu.
+- Have Node installed. Do this on Ubuntu using ```sudo apt-get install -y nodejs ``` and on Mac using homebrew  ```brew install node ```.
+- Have a Github account working with ssh connection, NOT https. For a guide on ssh setup, refer here: https://help.github.com/articles/generating-ssh-keys/
+- Fill out survey on what you want to get out of the class and your previous experience. The link to the survey is here [TODO]. 
+
+##Welcome to OlinJS! 
+We are excited to have you in this class and look forward to a great semester of learning Node.js together. To start off, here is your teaching team: 
+
+- Ben Kahle
+- Evan Dorsky
+- Evan Simpson
+- Thomas Nattestad
+
+Helping us this semester with Ninja duties are:
+- Joshe Langowitz
+- Sarah Walters
+
+##Syllabus
```

Comment:
This high level course stuff is all included in the main repository readme already.